app/routers/profile.py

```diff
@@ -23,18 +24,31 @@
 
 def get_placeholder_user():
     return User(
+        id=1,
```

Comment:
If id = 1 is busy, the program will crash. 
Recommends a variable boot at first startup and then retrieving the information from there.